.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ jobs:
   build:
     docker:
       # Primary container image where all steps run.
-      - image: avalonmediasystem/avalon:7.3.0-dev
+      - image: avalonmediasystem/avalon:7.5.0-dev-ruby3
```

Comment:
Once this PR is merged a new image will be built and pushed to docker hub by bamboo so I'll change this to that image in a subsequent PR.